### DISCUSSION TITLE
Welcome to return-youtube-dislike Discussions!

### DISCUSSION THREAD

**Discussion by Anarios** [Anarios/return-youtube-dislike#331] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CodePraisy** [comment]
I've just installed the chrome extension version and so far loving it. (it's sad that this even has to be created in the first place) I was wondering if this extension also adds a dislike to the public counter on videos when someone using the extension dislikes.
